config/software/cleanup.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright 2012-2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "cleanup"
```

Comment:
so i guess the only concern i've got is that i'd rather see us start doing this than not, and don't want to bikeshed on getting it into omnibus core, also think its not a bad approach to prototype this behavior in a software defn before adding it to core...